### ISSUE TITLE
Message Indicator (installed on High Sierra) interferes with FaceTime/Calls From iPhone notifications

### ISSUE THREAD

**Issue by Fofer** [dado3212/message-indicator#4]
It took me about a month to figure out, but I finally tracked down the craziest conflict. Shortly after I upgraded to High Sierra on my MacBook Pro, I noticed that whenever my iPhone would ring, the Continuity feature of macOS would launch Facetime, to alert me of the incoming call, but instead of merely showing me a notification like it used to in Sierra, it would instead turn on my webcam and open a videoconferencing window.  Every time.  Very annoying.  And so I turned off "Calls from iPhone" so this wouldn't happen anymore.  But I was bummed, because I really liked that Continuity feature, and being able to take voice calls on my MacBook Pro.

To cut to the chase, after lots and lots of troubleshooting and narrowing down the variables, I was able to pinpoint this SIMBL add-on, as the culprit. 

I know, this is crazy! But when Message Indicator is enabled, incoming calls open a FaceTime video on my Mac, and when Message Indicator is DISabled, incoming calls are displayed as regular notifications, as they should.  At least on my system. Could this be in conflict with another thing I have installed?  I dunno, but the "bug" and the fix are completely reproducible here.

Just thought I'd share, to potentially help someone else out there with the same issue.

**Comment by dado3212** [comment]
I have actually no idea why this happened.  I noticed that I was getting the same thing, so I removed and reinstalled all of my SIMBL tweaks, and it went away, despite the fact that the same ones were installed.  I don't think I can debug this, but I would try doing the same.  What SIMBL client are you using?

**Comment by Fofer** [comment]
I am using mySIMBL. When you say you "removed and reinstalled all of your SIMBL tweaks" what do you mean?  Disabled and re-enabled via your SIMBL client?  Deleted the files and re-downloaded fresh copies? Did you have to restart your Mac in between any of these steps?  

No need to debug further on my account, but I'm willing to try experimenting, if I could follow the same steps you did to make these functions coexist happily.  If not, I'll just leave Message Indicator disabled.  

It's especially odd, IMO, that this conflict only appeared after the upgrade to High Sierra.

**Comment by dado3212** [comment]
Deleted and the re-downloaded fresh copies (I actually downloaded the version from the Community Repo's, as opposed to mine).  I actually found that when I was deleting them it WASN'T Message Indicator but another SIMBL tweak that was causing it, which makes it seem like it's more like a SIMBL oddity than this particular one.  LMK if you have any success.  I didn't restart my Mac, but I did restart the programs (Dock and Messages) that were being altered by the packages.

**Comment by w0lfschild** [comment]
It could be because message indicator was trying to load into `com.apple.security.pboxd` and `com.apple.appkit.xpc.openAndSavePanelService` but this weekend when I was updating some of my repos I re-published `message-indicator version 1.1.0` where I removed those and just left `com.apple.iChat` ?

**Comment by Fofer** [comment]
Thanks for corroborating my report and shedding additional insight, I appreciate it.     I'll keep troubleshooting to try and learn more, and will share what I discover.

I'm wondering now if this same issue extends to my final mystery with High Sierra and iOS 11.2.  I can only get the new "Calls Handoff" to work between my iPhone and my iPad. I can't get it to work with my MacBook Pro.  My MacBook Pro does ring properly now with incoming calls, as mentioned above.  And I can accept the call.  I just can't get the new "Calls Handoff" feature to work, to allow me to transfer the call from the iPhone to the MacBook Pro after the call is already in progress.  And it should, according to these pages. I have the latest hardware and all the latest software updates.

http://www.iphoneincanada.ca/news/ios-11-2-handoff-phone-calls-to-mac/
http://www.redmondpie.com/how-to-handoff-a-call-from-iphone-to-ipad-or-mac-new-in-ios-11.2/
https://www.reddit.com/r/iphone/comments/7ialwi/underrated_ios_112_feature_calls_handoff/

I have disabled all of my SIMBL add-ons and the issue persists.  I tried completely uninstalling SIMBL to test further and that didn't work either, but I'm not entirely sure that I got every last piece of it.  The "Uninstall SIMBL" button in mySIMBL didn't seem to do anything, as best as I could tell.  At least when I rebooted, SIMBL came back.

Have either of you seen anything like this?
